metricbeat/module/linux/memory/data.go

```diff
@@ -71,14 +76,17 @@ func FetchLinuxMemStats(baseMap common.MapStr, hostfs resolve.Resolver) error {
 	if err != nil {
 		return errors.Wrap(err, "error getting huge pages")
 	}
-	thp["swap"] = common.MapStr{
-		"out": common.MapStr{
-			"pages":    vmstat.ThpSwpout,
-			"fallback": vmstat.ThpSwpoutFallback,
-		},
+	baseMap["hugepages"] = thp
```

Comment:
L88 is exactly the same. Is it meant to be duplicated?

Reply:
ooop, good catch!